### ISSUE TITLE
[Accessibility: Error Identification] mandatory field validation is missing in login page

### ISSUE THREAD

**Issue by atreyeemukhopadhyay** [kimchi-project/wok#72]
user should not be able to submit the login form without entering the credentials.
for wrong credentials username should be auto focused.

![login](https://cloud.githubusercontent.com/assets/13199397/14014447/759c4600-f1d8-11e5-8955-ad0ccaebd530.jpg)
